Remotion/ObjectBinding/Web.Development.WebTesting/ScreenshotCreation/ScreenshotBocDateTimeValuePickerExtensions.cs

```diff
@@ -172,30 +172,51 @@ private static int GetDayOfTheWeekIndex (DateTime date)
       return (int) date.DayOfWeek - 1;
     }
 
-    private static bool IsVisible (IFluentScreenshotElementWithCovariance<ScreenshotBocDateTimeValuePicker> fluentDatePicker)
+    private static void WaitUntilVisible (IFluentScreenshotElementWithCovariance<ScreenshotBocDateTimeValuePicker> fluentDatePicker, int timeout)

```

Comment:
I would prefer to not reorder items in the same commit as when changes are made. Often (not always and it usually depends on a lot of things) I also prefer not to restructure the class when doing just a bug fix that doesn't require me to actually do move methods around. If I do have to, rule 1 applies. Otherwise, reviewing the changes is quite difficult.